docs/_toc.yml

```diff
@@ -6,5 +6,6 @@ root: intro
 parts:
   - caption: Help & reference
     chapters:
+    - file: gnssa
```

Comment:
Right now this is in `Help & reference`. Maybe worth thinking about another `part`? Maybe a `Getting Started` section? Not sure... you can simply create another section by doing

```yaml
parts:
  - caption: Getting Started
     chapters:
     - file: gnssa

```